gooey/gui/controller.py

```diff
@@ -108,8 +171,8 @@ def required_section_complete(self):
   def success_dialog(self):
     self.show_dialog(i18n._("execution_finished"), i18n._('success_message'), wx.ICON_INFORMATION)
 
-  def error_dialog(self, error_msg):
-    self.show_dialog(i18n._('error_title'), i18n._('uh_oh').format(error_msg), wx.ICON_ERROR)

```

Comment:
@chriskiehl Gooey catches stderr to show python exceptions in a error dialog. It doesn't work as designed when wrapped program uses stderr to print messages to user and stdout to optional data out. Lots of unix tools work like this, for example, wget and dd. Also it solves buffered output from frozen programs as stdout is always unbuffered.